pkg/image/importer/dockerv1client/client.go

```diff
@@ -13,19 +13,71 @@ import (
 	"strings"
 	"time"
 
+	"github.com/docker/distribution/manifest/schema1"
+	"github.com/docker/distribution/manifest/schema2"
 	"github.com/fsouza/go-dockerclient"
 	"github.com/golang/glog"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/conversion"
 	knet "k8s.io/apimachinery/pkg/util/net"
 	"k8s.io/client-go/transport"
-
-	"github.com/docker/distribution/manifest/schema1"
-	"github.com/docker/distribution/manifest/schema2"
+	kapi "k8s.io/kubernetes/pkg/api"
 
 	imageapi "github.com/openshift/origin/pkg/image/apis/image"
 )
 
+// this is the only entrypoint which deals in github.com/fsouza/go-dockerclient.Image and expects to use our conversion capability to coerce an external
+// type into an api type.  Localize the crazy here.
+// TODO this scheme needs to be configurable or we're going to end up with weird problems.
+func init() {
+	err := kapi.Scheme.AddConversionFuncs(
```

Comment:
spoke with @smarterclayton in person, this is ok to start the split.  When we fix the scheme problem, we should fix this.